### ISSUE TITLE
[Idea] Import of DataInterfaces from local modules or public non-NeuroConv host

### ISSUE THREAD

**Issue by CodyCBakerPhD** [NeurodataWithoutBorders/nwb-guide#184]
Original thoughts on the subject of allowing users to import and run any code from the internet was that it might give way to security issues

But I have a really hard time seeing how the GUIDE could be used for performing an entire conversion without the ability to use custom interfaces, especially for behavior streams that might be experiment-specific

One idea was to setup a publication pipeline similar to how we manage extensions; we could allow running of interfaces that have been 'signed off' by a maintainer of that pipeline, that is confirmed to not have any malicious code (though IDK if I feel totally comfortable signing off with that level of responsibility hanging on us lol)

We could also get around the responsibility issue by having the user confirm, via several scary confirmation boxes and maybe an e-signature, that they understand the risks of downloading and running non-NeuroConv code in their application environment; just need to make it clear that they can't blame the NWB team in the event something bad were to happen there

TLDR; I wrote a simple custom behavior interface on my local system and I want to use it in my GUIDE pipeline. But the GUIDE only lists NeuroConv supported interfaces

**Comment by CodyCBakerPhD** [comment]
This isn't working super well outside of dev mode

Looking into some other ideas but for now it's pretty compllicated
